samples/openthread/cli/prj.conf

```diff
@@ -4,7 +4,6 @@
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 CONFIG_NEWLIB_LIBC=y
-CONFIG_ENTROPY_CC310=n
```

Comment:
So are we good with CC310 entropy now? It was disabled not long ago.

Should we remove the DTS overlays as well? Or do we want to keep both, RNG and CC310 entropy?

Reply:
No, dts overlay alone fixes the entropy issue with ping. However we can not disable cc310 entropy entirely by setting CONFIG_ENTROPY_CC310=n as this makes nrf_security not compile.